TM1py/Services/CellService.py

```diff
@@ -540,10 +590,14 @@ def extract_cellset(self, cellset_id, cell_properties=None, top=None):
             cell_properties = ['Value', 'Ordinal']
         elif 'Ordinal' not in cell_properties:
             cell_properties.append('Ordinal')
-        raw_cellset = self.extract_cellset_raw(cellset_id, cell_properties=cell_properties, top=top)
-        return Utils.build_content_from_cellset(raw_cellset_as_dict=raw_cellset,
-                                                cell_properties=cell_properties,
-                                                top=top)
+
+        raw_cellset = self.extract_cellset_raw(
+                cellset_id,
+                cell_properties=cell_properties,
+                elem_properties=['UniqueName'],
```

Comment:
I think in 597 and 598 we need to pass elem_properties and member_properties variables

Reply:
Hi Marius,
Both elem_properties and member_properties are being passed. Do you mean not as a an explicit list of only 'UniqueName' ?

Reply:
Please ignore my previous comment on this one. It's wrong.